docker/utils/socket.py

```diff
@@ -84,17 +84,4 @@ def frames_iter(socket):
                 # We have reached EOF
                 return
             n -= data_length
-            yield result
-
-
-def socket_raw_iter(socket):
```

Comment:
doh. I was wondering what was the difference with `frames_iter`. Now that I broken `tty=True` is broken I understand why it's there... Will back out that change

Reply:
Fixed:


```python
In [6]: res = container.exec_run(cmd, stream=True, demux=False, tty=True) 
                                                                          
In [7]: next(res.output)                                                  
Out[7]: b'hello stdout\r\nhello stderr\r\n'                               
                                                                          
In [10]: res = container.exec_run(cmd, stream=True, demux=True, tty=True)         
                                                                                  
In [11]: next(res.output)                                                         
Out[11]: (None, b'hello stdout\r\nhello stderr\r\n', None)                        
```